package.json

```diff
@@ -27,6 +27,7 @@
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
+    "@embroider/macros": ">= 0.48.1 < 2.0.0-alpha.1",
```

Comment:
`@embroider/macros` is "stable", this ensures that we can continually get the most recent version (and don't force an app to end up with a bunch of versions due to our dependency).